Main.java

```diff
@@ -3,58 +3,51 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.sql.*;
 import java.util.StringTokenizer;
+import com.company.dbconnection.DatabaseLink;
 
 public class Main {
 
-    static final String FILE_TYPE_TXT = ".txt";
-    static final String FILE_TYPE_JSON = ".json";
-    static final String REMOVE_SPECIAL_CHAR = "[^a-zA-Z0-9 -]";
-    static final String SINGLE_SPACE = "";
-    static String searchResult;
+    static String textFilePath = null;

```

Comment:
These should not be fields, you can change them to local variables and pass them as parameters

Reply:
I have changed them to local variable.